Remotion/Web/Core/Res/Themes/NovaViso/HTML/DropDownMenu.css

```diff
@@ -24,6 +24,42 @@ span.DropDownMenuSelect
   cursor: pointer;
 }
 
+span.DropDownMenuContainer.primary span.DropDownMenuSelect:not(.disabled)
+{
+  background-color: var(--color-primary);
+  border-color: var(--color-primary);
+}
+
+span.DropDownMenuContainer.primary span.DropDownMenuSelect:not(.disabled) a.DropDownMenuLabel
+{
+  color: var(--color-primary-text);
+  border-right-color: var(--color-primary-text);
+}
+
+span.DropDownMenuContainer.primary span.DropDownMenuSelect:not(.disabled):hover,
+span.DropDownMenuContainer.primary span.DropDownMenuSelect:not(.disabled):focus,
+span.DropDownMenuContainer.primary span.DropDownMenuSelect:not(.disabled).focus
+{
+  background-color: var(--color-primary-highlight);
+  border-color: var(--color-primary-highlight);
+}
+
+span.DropDownMenuContainer.primary span.DropDownMenuSelect:not(.disabled) a.DropDownMenuButton img
+{
+  filter: invert();
+}
```

Comment:
What I mentioned in the other chat, if filter in a css file is the only way, we stick with doing this in the svg file via duplication.

What I was trying to achive was applying color-styling (but no filters) directly inside the svg file to separate the shape from the color, since the svg-file already has a layered approach for the chevron (declaring the shape and using the shape). Since we can't colorize the shape inside the svg file at the using-place, we stick with duplication.